Makefile

```diff
@@ -20,7 +20,7 @@ IMAGE_PREFIX ?= deis
 # Docker Root FS
 BINDIR := ./rootfs
 
-DEIS_REGISTRY ?= ${DEV_REGISTRY}/
+DEIS_REGISTRY ?= quay.io/
```

Comment:
It might be better to specify $DEV_REGISTRY as an environment variable in .travis.yml instead. That way we don't have hardcoded registries in the Makefile. DEV_REGISTRY is a holdover from v1 :)